pytest_rts/tests/test_db.py

```diff
@@ -86,3 +86,60 @@ def test_update_db_from_test_mapping():
         new_func_linenumbers = new_func_line_dict[key]
         assert old_func_linenumbers[0] + shift == new_func_linenumbers[0]
         assert old_func_linenumbers[1] + shift == new_func_linenumbers[1]
+
+
+def test_add_new_tests():
+    new_tests = {"tests/test_car.py::test_acceleration"}
+
+    db = DatabaseHelper()
+    db.init_conn()
+    db.add_new_tests(new_tests)
+    db.close_conn()
+
+    conn = sqlite3.connect(DB_FILE_NAME)
```

Comment:
https://docs.python.org/3.6/library/sqlite3.html#using-the-connection-as-a-context-manager